scheduler/integration-test/src/scheduler.test.js

```diff
@@ -99,6 +104,19 @@ describe('Scheduler', () => {
   })
 })
 
+async function receiveAmqp (url, exchange, topic, queue) {
+  const connection = await amqp.connect(url)
+  const channel = await connection.createChannel()
+  await channel.assertExchange(exchange, 'topic', {durable: true})
+  const q = await channel.assertQueue(queue)
+  await channel.bindQueue(q.queue, exchange, topic)
+  await channel.consume(q.queue, consumeEvent)
```

Comment:
You can look into PR #128 how I solved the AMQP connection in the integration tests....
Important is to also close the connection, otherwise Jest won't shutdown ;)